android/src/test/io/grpc/android/AndroidChannelBuilderTest.java

```diff
@@ -0,0 +1,25 @@
+package io.grpc.android;
+
+import static org.junit.Assert.assertNotNull;
+
+import io.grpc.ChannelImpl;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Tests for {@link io.grpc.android.AndroidChannelBuilder}.
+ */
+@RunWith(JUnit4.class)
+public class AndroidChannelBuilderTest {
+
+  @Test
+  public void testDefaultToOkHttp() {
```

Comment:
Renamed test. The goal is simply to ensure that some channel is properly produced.